cmd/forwarder.go

```diff
@@ -120,6 +119,20 @@ func Forwarder() *cobra.Command {
 					})
 				}
 			}()
+
+			// listen for signals so can gracefully stop kakfa producer
+			go func() {
+				for _ = range signals {
```

Comment:
I'd rather just see `<-signals` here instead of a for loop since this loop always calls `os.Exit` at the end. A "loop" is kind of misleading. We're really just waiting for the first item on the signals channel.